pkg/testing/fixture.go

```diff
@@ -916,3 +923,43 @@ type AgentInspectOutput struct {
 		Data string `yaml:"data"`
 	} `yaml:"signed"`
 }
+
+func decreasePackageVersionFor8_11(t *testing.T, f *Fixture) {
+	installFS := os.DirFS(f.WorkDir())
+	var matches []string
+
+	err := fs.WalkDir(installFS, ".", func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if d.Name() == agentVersion.PackageVersionFileName {
+			matches = append(matches, path)
+		}
+		return nil
+	})
+	require.NoError(t, err)
+
+	t.Logf("package version files found: %v", matches)
+
+	for _, m := range matches {
+		versionFile := filepath.Join(f.WorkDir(), m)
+		fmt.Printf("removing package version file %q\n", versionFile)
+
+		pv, err := version.ParseVersion(f.version)
+		if err != nil {
+			require.NoError(t, err, "could not parse fixture version")
+		}
+		if pv.Major() == 8 && pv.Minor() > 11 {
```

Comment:
```suggestion
		if pv.Major() == 8 && pv.Minor() == 11 {
```

Shouldn't this be an equals?

Reply:
yes, I fixed here already